projects/dsp-ui/src/lib/viewer/representation/upload-form/upload-form.component.scss

```diff
@@ -0,0 +1,101 @@
+// overall styles
+.form-container {
+  margin: 20px auto;
+  width: 50%;
+}
+.full-width {
+  width: 100%;
+}
+button {
+  cursor: pointer;
+}
```

Comment:
You could also use our global css class "link" on the button in the html template. But normally a button has already cursor type `pointer`.

Reply:
I was not aware of global styles. Cleaned a bit up those buttons with f3b60c2.

Reply:
> I was not aware of global styles. 

Yes sure. This is something we have to document better